src/root/utils/get-dtype-meta.js

```diff
@@ -1,6 +1,10 @@
-import { disasterType } from './field-report-constants';
+import store from './store';
```

Comment:
@frozenhelium @batpad Really afraid of this, but couldn't really resolve it any other way without rewriting the whole thing.

Reply:
Can we get the places that call `getDtypeMeta` to send in `disasterTypes` as a parameter to the function, so the function does not need to call the state directly, but just receives it as parameter - and I'm assuming the view or wherever `getDtypeMeta` is called from already has access to this from the state and can send it down as a param? Happy to look at this together - I agree that we should avoid accessing the store directly here if possible. Curious to see if @frozenhelium has a suggestion.

Reply:
Yep, I think we should pass in the `disasterTypes` as argument

Reply:
@frozenhelium @batpad I was able to completely remove the function alltogether.